proof/invariant-abstract/Finalise_AI.thy

```diff
@@ -1579,4 +1579,11 @@ crunches test_reschedule, tcb_release_remove
   and obj_at[wp]: "\<lambda>s. P (obj_at Q p s)"
   (wp: crunch_wps simp: crunch_simps)
 
+lemma tcb_ep_find_index_inv [wp]:
```

Comment:
Nitpick: not sure if there are any strict rules on this, but almost all attributes immediately follow the name of the lemma